agent/taskresource/types/types.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/aws/amazon-ecs-agent/agent/taskresource"
 	asmauthres "github.com/aws/amazon-ecs-agent/agent/taskresource/asmauth"
+	asmsecretres "github.com/aws/amazon-ecs-agent/agent/taskresource/asmsecret"
```

Comment:
typo: `asmsecrets "github...`

Reply:
Since all others follow the naming convention ***res, so I use **asmsecretres**